### PR TITLE
also ignore Num_Lock state

### PR DESCRIPTION
**Pull request by buxit**
similar to Caps_Lock also ignore Num_Lock state

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ function destroyOverlay() {
 function onHotkeyPressed() {
   Log.debug("Hot key pressed");
   //Clear the lock bit so the status of Caps_Lock won't affect the functionality
-  let multiKeysCode = keymap.get_modifier_state() & (~2);
+  let multiKeysCode = keymap.get_modifier_state() & (~(2 | 16));
```
Comment (pbxqdown) [inline extension.js:131]: I nearly forgot how mask works.. I think 2 is capslock and 16 is NumsLock , so doing a "bitwise or" will mask both 2 and 16?

Comment (igolskyi) [reply extension.js:131]: Ok, I will test it.

Comment (igolskyi) [reply extension.js:131]: image
@pbxqdown I've tested this suggestion. It works.

Comment (igolskyi) [reply extension.js:131]: P.S. But there is one point. When I changed the code and restarted gnome-shell `Alt + F2 -> r`. Code 8 appeared in the logs when the Alt key was pressed. However, the old number 24 remained in the extension settings window, which I had to change manually to 8.

Comment (pbxqdown) [reply extension.js:131]: Nice catch! This line needs to change as well: https://github.com/pbxqdown/gnome-shell-extension-transparent-window/blob/e4261bd9b47adcb9863c7ef11b6e46a89a33eaa4/prefs.js#L48

Comment (igolskyi) [reply extension.js:131]: Will be better to move these two places to one function in utils.js.
For example, to create a function 
```js
// utils.js
function getMultiKeysCode() {
    keymap.get_modifier_state() & (~(2 | 16));
}
```
and then import it into two places, as isVersionGreaterOrEqual() for example.

Comment (igolskyi) [reply extension.js:131]: @pbxqdown let's wait until the weekend. If it will not be done, I can do it.

Comment (pbxqdown) [reply extension.js:131]: @igolskyi Sure, thanks for helping with this!

Comment (igolskyi) [reply extension.js:131]: @pbxqdown I've done it here https://github.com/pbxqdown/gnome-shell-extension-transparent-window/pull/20. Please review it.

**Comment by pbxqdown** [comment]
Already resolved in #20. Thanks for bringing up this issue!

**Pull request closed without merging**